index.html

```diff
@@ -43,6 +43,153 @@ <h3>LET'S CONNECT</h3>
 
         </section>
     </div>
+    <div id="working">
+        <div class="project">
+            <img class="project_image" src="assets/Snapshoot Portfolio.png" alt="">
+            <div class="general">
+                <h3>Tonic</h3>
+                <div class="general_inf">
+                    <p>Canopy.Back-end.2023</p>
+                </div>
+                <p class="project_description">A daily selection of privately personalized reads; no 
+                    accounts or sign-ups required.</p>
+                <footer>
+                    <ul class="skills">
+                        <li>Html</li>
+                        <li>Css</li>
+                        <li>Javascript</li>
+                    </ul>
+                    <div class="external_link">
+                        <a href="#">see project</a>
+                    </div>
+                </footer>
+            </div>
+        </div>
+        <div class="project">
+            <img class="project_image" src="assets/Snapshoot Portfolio (1).png" alt="">
+            <div class="general">
+                <h3>Multi-post Stories</h3>
+                <div class="general_inf">
+                    <p>Canopy.Back-end.2023</p>
+                </div>
+                <p class="project_description">A daily selection of privately personalized reads; no 
+                    accounts or sign-ups required.</p>
+                <footer>
+                    <ul class="skills">
+                        <li>Html</li>
+                        <li>Css</li>
+                        <li>Javascript</li>
+                    </ul>
+                    <div class="external_link">
+                        <a href="#">see project</a>
+                    </div>
+                </footer>
+            </div>
+        </div>
+        <div class="project">
+            <img class="project_image" src="assets/Snapshoot Portfolio (2).png" alt="">
+            <div class="general">
+                <h3>Tonic</h3>
+                <div class="general_inf">
+                    <p>Canopy.Back-end.2023</p>
+                </div>
+                <p class="project_description">A daily selection of privately personalized reads; no 
+                    accounts or sign-ups required.</p>
+                <footer>
+                    <ul class="skills">
+                        <li>Html</li>
+                        <li>Css</li>
+                        <li>Javascript</li>
+                    </ul>
+                    <div class="external_link">
+                        <a href="#">see project</a>
+                    </div>
+                </footer>
+            </div>
+        </div>
+        <div class="project">
```

Comment:
Kindly update the styling of each of your cards to better match the figma design🙏.
Your design:
<img width="232" alt="image" src="https://user-images.githubusercontent.com/37116322/219722752-c434d506-ff75-4c6c-b1c8-8cea5d76a018.png">

Figma design:
<img width="227" alt="image" src="https://user-images.githubusercontent.com/37116322/219483113-557ca771-0e25-4c35-8fa0-b8284d450cb3.png">

A few things to note:

<img width="201" alt="image" src="https://user-images.githubusercontent.com/37116322/219723417-1d7bfa54-00f6-42cc-a27e-7926b7ba8a1c.png">

- Your design is currently missing the separator in between the list items in the screenshot above. Please add it and also update the color of the text to better match the figma design
- There is a lot of unnecessary space in between the paragraph text. Please update it
- The technologies (HTML, CSS, js) currently have too much left and right padding
- The See project button should be aligned to the left, should have a bigger font size and should have equal padding on all sides

**Please try your best to make sure that your design matches the Figma design as much as possible😎**

Reply:
thank you for notifying the error.
 1) separator is added
 2) the unnecessary space is been removed
 3) the see project button is aligned to the left and bigger font is added.